sdk/keyvault/Azure.Security.KeyVault.Administration/src/Azure.Security.KeyVault.Administration.csproj

```diff
@@ -36,6 +37,4 @@
     <Compile Include="$(AzureCoreSharedSources)Base64Url.cs" LinkBase="Shared\Core" />
   </ItemGroup>
 
-  <!-- Import the Azure.Core project -->
-  <Import Project="$(MSBuildThisFileDirectory)..\..\..\core\Azure.Core\src\Azure.Core.props" />
```

Comment:
Does this mean that any new features on which we need to depend need to be vetted and published before we can even use them? That seems overly prohibitive and will increase development time. Could we instead consider passing a variable from release runs that automatically swaps a ProjectRef to a PackageRef or something similar?